install/updateVersion.sh

```diff
@@ -117,39 +117,44 @@ function check_git_status() {
 function merge_files() {
   SRC=$TEMP_DIR/templates
   DEST=$ROOT/install/kubernetes
+
+  # istio.yaml and istio-auth.yaml file contain cluster-wide installations
   ISTIO=$DEST/istio.yaml
-  ISTIO_CLUSTER_WIDE=$DEST/istio-cluster-wide.yaml
   ISTIO_AUTH=$DEST/istio-auth.yaml
+  ISTIO_ONE_NAMESPACE=$DEST/istio-one-namespace.yaml
+  ISTIO_ONE_NAMESPACE_AUTH=$DEST/istio-one-namespace-auth.yaml
   ISTIO_INITIALIZER=$DEST/istio-initializer.yaml
 
-#TODO remove 2 lines below once the e2e tests no longer look for this file
-  cp $SRC/istio-rbac-beta.yaml.tmpl $DEST/istio-rbac-beta.yaml
+  # TODO remove 3 lines below once the e2e tests no longer look for this file
+  echo "# GENERATED FILE. Use with Kubernetes 1.7+" > $DEST/istio-rbac-beta.yaml
+  echo "# TO UPDATE, modify files in install/kubernetes/templates and run install/updateVersion.sh" >> $DEST/istio-rbac-beta.yaml
+  cat $SRC/istio-rbac-beta.yaml.tmpl >> $DEST/istio-rbac-beta.yaml
+
 
   echo "# GENERATED FILE. Use with Kubernetes 1.7+" > $ISTIO
   echo "# TO UPDATE, modify files in install/kubernetes/templates and run install/updateVersion.sh" >> $ISTIO
   cat $SRC/istio-ns.yaml.tmpl >> $ISTIO
   cat $SRC/istio-rbac-beta.yaml.tmpl >> $ISTIO
   cat $SRC/istio-mixer.yaml.tmpl >> $ISTIO
+  cat $SRC/istio-config.yaml.tmpl >> $ISTIO
```

Comment:
Do we also need to update those templates as well by adding the info of `GENERATED FILE. Use with Kubernetes 1.7+`, @andraxylia

Reply:
Not sure what you mean... the templates are not generated.

Reply:
@gyliu513 see line 134,135 ?

Reply:
thanks, got it, then seems this is missing for the $ISTIO_INITIALIZER?

Reply:
Added it, ptal.